### PR TITLE
Added section and cleared up documentation in getting started

### PR DESCRIPTION
**Pull request by lpliger**
As a new user that's not very experienced with programming I found the documentation to be a bit unclear, especially regarding the "getting started" section. So I made my best attempt to clear it up in a way that could be understood by people with less experience such as myself. Also there was a missing link :)

### CODE REVIEW

**Review by AlbertDeFusco — APPROVED** [review]
I like it, thanks!

File: docs/tutorials/getting-started.md [hunk 1]
```diff
@@ -5,16 +5,50 @@ This page will guide you through getting started with PyScript.
...
-than a web browser. We recommend using [Chrome](https://www.google.com/chrome/).
+than a web browser (we recommend using [Chrome](https://www.google.com/chrome/)) and a text editor, even though using your [IDE](https://en.wikipedia.org/wiki/Integrated_development_environment) of choice might be convenient.
 
 If you're using [VSCode](https://code.visualstudio.com/), the
 [Live Server extension](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer)
 can be used to reload the page as you edit the HTML file.
 
+## Trying before installing
+
+If you're new to programming and know nothing about HTML or just want to try some of PyScript features, we recommend using the [](https://en.wikipedia.org/wiki/Read%E2%80%93eval%E2%80%93print_loop) [PyScript REPL example](https://pyscript.net/examples/repl.html) instead so you can have a programming experience in a REPL that doesn't require any setup. This REPL can be used to have an interactive experience using Python directly.
```
Comment (marimeireles) [inline docs/tutorials/getting-started.md:16]: >`[](https://en.wikipedia.org/wiki/Read%E2%80%93eval%E2%80%93print_loop)`

Is this a typo?

**Review by marimeireles — COMMENTED** [review]
Looks good to me =)
Thanks for laying out such a great explanation!
Just one question and we can merge it IMO

File: docs/tutorials/getting-started.md (hunk 1 above)
Comment (lpliger) [reply docs/tutorials/getting-started.md:16]: Yeah, I meant to link the word "REPL" to Wikipedia, I'll fix it.

Comment (marimeireles) [reply docs/tutorials/getting-started.md]: I see, cool!

**Comment by marimeireles** [comment]
Thanks for the contribution @lpliger :)

**Pull request merged**